zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -324,12 +328,17 @@ where
 
                 min_time: 0,
 
-                mutable: vec![],
+                // Hardcoded list of block fields the miner is allowed to change.
+                mutable: constants::GET_BLOCK_TEMPLATE_MUTABLE_FIELD.to_vec(),
+
+                // A range of valid nonces that goes from `u32::MIN` to `u32::MAX`.
+                nonce_range: constants::GET_BLOCK_TEMPLATE_NONCE_RANGE_FIELD.to_string(),
 
-                nonce_range: empty_string.clone(),
+                // Max legacy signature operations in the block.
+                sigop_limit: MAX_BLOCK_SIGOPS,
 
-                sigop_limit: 0,
-                size_limit: 0,
+                // Max block size in bytes
+                size_limit: MAX_BLOCK_BYTES,
```

Comment:
Can you add documentation to the `GetBlockTemplate` struct for these new fields?

Reply:
https://github.com/ZcashFoundation/zebra/pull/5558/commits/5590d0cab8339748ed97f88fcf40cf144916ec18

Reply:
My suggestionPR documented the roots fields that I missed in a previous PR.

But it's also important to document the mutable, nonce_range, sigop_limit, and size_limit fields on the struct. (The docs here are good, you can just copy them to the struct definition.)